File: Lab3/Skates.java
package ua.lviv.iot.hockeyclub;

public class Skates extends Good {
	private String sizeOfLeg;

	public Skates() {

	}

	public Skates(GoodType goodType, GoodName name, int price, String material, double weight, String manfucturer,
			String color, int amount, String sizeOfLeg) {

		super(goodType, name, price, material, weight, manfucturer, color, amount);

		this.sizeOfLeg = sizeOfLeg;
	}

	public String getSizeOfLeg() {
		return sizeOfLeg;
	}

	public void setSizeOfLeg(String sizeOfLeg) {
		this.sizeOfLeg = sizeOfLeg;
	}

}

File: Python lab2/goods/Skates.py
from goods.Good import *
from enums.GoodType import *
from enums.GoodName import *


class Skates(Good):
    goods_type = GoodType.FORANY
    goods_name = GoodName.SKATES

    def __init__(self, sizeOfLeg, name, price, material, weight, manufacturer, color, amount):
        self.sizeofleg = sizeOfLeg
        Good.__init__(self, name, price, material, weight, manufacturer, color, amount)

    def __str__(self):
        return "Name: " + str(self.name) + " price: " + str(self.price) + " the size of leg is " + str(self.sizeofleg)